diagnostics-app/src/app/telemetry/telemetry-card/telemetry-card-content/telemetry-card-content.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="telemetry-card-content" *ngFor="let item of $any(info) | keyvalue: originalOrder">
+    <b>{{ item.key }}</b>:
+    <ng-container *ngIf="returnType(item.value) !== 'object'; else printObject">
```

Comment:
Random idea: When will `item.value` != `object`? If it is loading, maybe we can create a loading template for the loading items instead of skipping them.

Reply:
If the item.value is not an object, it displays the value directly.